po/pt_BR.po

```diff
@@ -0,0 +1,453 @@
+# Portuguese translations for solaar package.
+# Copyright (C) 2015 THE solaar'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the solaar package.
+# Automatically generated, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: solaar 0.9.2\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-10-08 17:47-0300\n"
+"PO-Revision-Date: 2015-10-08 17:47-0300\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Sídnei A. Drovetto Jr.\n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "critical"
+msgstr "crítica"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "empty"
+msgstr "vazia"
+
+#: lib/logitech_receiver/i18n.py:38 lib/logitech_receiver/i18n.py:41
+msgid "full"
+msgstr "cheia"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "good"
+msgstr "boa"
+
+#: lib/logitech_receiver/i18n.py:38
+msgid "low"
+msgstr "baixa"
+
+#: lib/logitech_receiver/i18n.py:41
+msgid "almost full"
+msgstr "quase cheia"
+
+#: lib/logitech_receiver/i18n.py:41
+msgid "discharging"
+msgstr "descarregando"
+
+#: lib/logitech_receiver/i18n.py:41
+msgid "recharging"
+msgstr "recarregando"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "invalid battery"
+msgstr "bateria inválida"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "slow recharge"
+msgstr "recarga lenta"
+
+#: lib/logitech_receiver/i18n.py:42
+msgid "thermal error"
+msgstr "erro térmico"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "device not supported"
+msgstr "dispositivo não suportado"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "device timeout"
+msgstr "tempo esgotado do dispositivo"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "sequence timeout"
+msgstr "tempo esgotado da sequência"
+
+#: lib/logitech_receiver/i18n.py:45
+msgid "too many devices"
+msgstr "dispositivos demais"
+
+#: lib/logitech_receiver/i18n.py:48
+msgid "Bootloader"
+msgstr "Bootloader"
+
+#: lib/logitech_receiver/i18n.py:48 lib/solaar/ui/window.py:535
+msgid "Firmware"
+msgstr "Firmware"
+
+#: lib/logitech_receiver/i18n.py:48
+msgid "Hardware"
+msgstr "Hardware"
+
+#: lib/logitech_receiver/i18n.py:48
+msgid "Other"
+msgstr "Outro"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "closed"
+msgstr "fechado"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "open"
+msgstr "aberto"
+
+#: lib/logitech_receiver/notifications.py:67
+msgid "pairing lock is "
+msgstr "trava de emparelhamento está "
+
+#: lib/logitech_receiver/notifications.py:150 lib/solaar/ui/notify.py:118
+msgid "unpaired"
+msgstr "desemparelhar"
```

Comment:
This should be "desemparelhado" since "unpaired" is past tense. Unpair -> "desemparelhar"